src/Symfony/Component/Messenger/Asynchronous/Middleware/SendMessageMiddleware.php

```diff
@@ -36,12 +35,11 @@ public function __construct(SenderLocatorInterface $senderLocator, array $messag
     /**
      * {@inheritdoc}
      */
-    public function handle($message, callable $next)
+    public function handle($envelope, callable $next)
```

Comment:
Can you add a `@param Envelope $envelope` above the `{@inheritdoc}`? I think that this would be better to explicit it's an Envelope.